src/containers/Assign/AssignmentSectionRow.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react'
```

Comment:
It's hard to tell what changed in this file, as the diff just shows this as all new, and the other one deleted, rather than a moved file with diff.

Can you just give a 1-sentence summary of the changes in this file?

Reply:
Just added the logic for when `re-assign` is selected in the dropdown a new component is displayed to select the new reviewer